src/app/server/app.go

```diff
@@ -40,13 +44,17 @@ func NewApp(opts ...AppOptions) *App {
 	// in config constants
 	conf.Env()
 
-	// Set up gin
+	// Make an engine
+	engine := echo.New()
+
+	// Set up echo
 	if !conf.UBool("debug") {
-		gin.SetMode(gin.ReleaseMode)
+		engine.SetDebug(true)
```

Comment:
I think more efficient way to do it with Echo is `engine.SetDebug(conf.UBool("debug"))`.